tycho-integration-test/src/statistics.rs

```diff
@@ -0,0 +1,217 @@
+use std::collections::{HashMap, HashSet};
+
+use tycho_test::execution::models::TychoExecutionResult;
+
+#[derive(Default, Clone)]
+pub struct TestStatistics {
+    pub blocks_processed: u64,
+    pub total_simulations: u64,
+    pub successful_simulations: u64,
+    pub failed_simulations: u64,
+    pub reverted_simulations: u64,
+    pub setup_failures: u64,
+    pub validation_passed: u64,
+    pub validation_failed: u64,
+    pub get_limits_success: u64,
+    pub get_limits_failure: u64,
+    pub get_amount_out_success: u64,
+    pub get_amount_out_failure: u64,
+    pub protocol_stats: HashMap<String, ProtocolStats>,
+    pub blocks_seen: HashSet<u64>,
+    pub unique_pools_tested: HashSet<String>,
+    pub slippage_values: Vec<f64>,
+}
+
+#[derive(Default, Clone)]
+pub struct ProtocolStats {
```

Comment:
Stats or Statistics?

Reply:
but it is not consistent across the several structs (at least it wasn't when I reviewed)